cirq/ops/common_gates.py

```diff
@@ -263,6 +283,16 @@ def _eigen_components(self):
             (1, np.diag([0, 1])),
         ]
 
+    def _pauli_expansion_(self) -> Dict[str, complex]:
+        if protocols.is_parameterized(self):
+            return NotImplemented
+        phase = 1j**(2 * self._exponent * (self._global_shift + 0.5))
+        angle = np.pi * self._exponent / 2
+        return {
+            'I': phase * np.cos(angle),
+            'Z': -1j * phase * np.sin(angle),
```

Comment:
How crazy would it be to use `cirq.Z` instead of the string `'Z'`...?

Reply:
Not too crazy IMO. I considered it and deferred it until we have tensor product since this is needed to create keys for multi-qubit expansions (e.g. II, XX, YY and ZZ for swap gates).